### PR TITLE
Develop contact section

### PR DESCRIPTION
**Pull request by zainsadaqat**
# Develop Contact Section with HTML Forms

The form includes the following fields:
* Full Name
* Email
* Message
* Submit Button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,5 +231,49 @@ <h3 class="language-name">CSS</h3>
...
+          </li>
+          <li>
+            <textarea
+              rows="4"
+              cols="50"
+              name="message"
+              placeholder="Write your message here"
+              maxlength="500"
+              class="input-textarea"
+            ></textarea>
```
Comment (george-swift) [inline index.html]: - [x] You integrated with the Formspree service correctly. However, the data entered into a form needs to be in the right format and certain fields need to be filled in order to effectively use the submitted form. Currently, only the name input field is `required`. Kindly add the `required` attribute to the `email` and `textarea` elements. All three fields are required for server processing.

File: index.html [hunk 2]
```diff
@@ -231,5 +231,49 @@ <h3 class="language-name">CSS</h3>
...
+          </li>
+          <li>
+            <button type="submit" class="contact-button">Get It Touch</button>
+          </li>
```
Comment (george-swift) [inline index.html:276]: - [x] The button submits the form 👍🏽 but buttons are multi-state objects and it's very important for users to get some visual feedback for a given interaction. The Figma template provided styling rules for `hover`, `active (when it's pressed)`, and `enabled (default state)`. Kindly implement this requirement.

File: style.css [hunk 3]
```diff
@@ -68,6 +92,53 @@ body {
...
+}
+
+.input {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  padding: 15px;
+  margin-bottom: 20px;
+  color: #172b4d;
+  font-weight: 600;
```
Comment (george-swift) [inline style.css]: [OPTIONAL]
- [x] The form implements flexbox correctly 👏🏽 . However, it's also important for its elements to adhere to the font style and weight from Figma as much as possible. It'll improve user experience as well.

**Review by george-swift — CHANGES_REQUESTED** [review]
### Status: Changes Required 🔔 

Hello Zain 👋🏽 ,

Good job so far! 👏🏽 👏🏽  You have implemented some of the basic requirements for this project. Your form looks great!  Before the next project, however, there are some issues you need to work on. But you are almost there! 🚀 

## Required Changes ♻️
### _Check the comments under the review._ 👇🏽 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏🏽 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @zainsadaqat 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.

May I highlight the following:
- The repo looks professional:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have a descriptive PR:heavy_check_mark:
- Your contact form is working properly:heavy_check_mark:

## Optional suggestion
  * Kindly, include the live link demo in your Readme file.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (zainsadaqat) [reply index.html]: Alright man

**Comment by zainsadaqat** [comment]
# Thank you, @Emmyn5600 for approving this pull request, and @george-swift for your useful tips and suggestions!

**Pull request merged**